app/views/shared/_header.html.slim

```diff
@@ -2,6 +2,11 @@ nav.navbar.navbar-light.bg-light
   h1.logo Fifteenglish
   form.form-inline
     - if user_signed_in?
+      .dropdown
+        .btn.btn-outline-success == link_to = current_user.email
+        .dropdown-content
+          == link_to 'Profile', current_user
+          == link_to 'Logout', destroy_user_session_path, method: :delete
```

Comment:
Rename link to 'Sign out', because we have other links with word 'Sign' (Sign in, Sign up)
And delete useless button on line 10